### PR TITLE
Adding Success Screen + microchanges

### PR DESCRIPTION
**Pull request by JimBratsos**
Adding Success Screen folder, which contains the JS file, the SCSS and the components folder which is a folder with all the SVG used in the process. Also tweaked some values in SCSS for landing page without any significant results ( I forgot to revert them back to normal, but no side effects come from it ). Finally I fixed a typo in landing page. Thanks

### CODE REVIEW

**Comment by ArnasDickus** [comment]
@JimBratsos There was a conflict in app.js regarding, comments, so I fix it for you.

File: src/components/LandingPage/sections/SectionOne/SectionOne.js
Comment (ArnasDickus) [inline src/components/LandingPage/sections/SectionOne/SectionOne.js:39]: Thanks for fixing spelling mistake.

File: src/components/LandingPage/LandingPage.module.scss
Comment (ArnasDickus) [inline src/components/LandingPage/LandingPage.module.scss]: vh is based on %. So 100vh is equal 100% of viewport height. There's no reason to put it at 500vh [Documentation](https://www.sitepoint.com/css-viewport-units-quick-start/).

File: src/components/Success/Success.module.scss
Comment (ArnasDickus) [inline src/components/Success/Success.module.scss]: Could you use rem or em instead of px, for accessibility?

File: src/components/Success/Success.js
Comment (ArnasDickus) [inline src/components/Success/Success.js]: I don't think you should be using SVG instead of text. Although the only argument I have is that Google robot can't read text on SVG.  So if you want to defend your choice. I'm all ears.

File: src/components/Success/Success.module.scss
Comment (ArnasDickus) [inline src/components/Success/Success.module.scss]: font-style and font-weight are normal by default. So you need to remove it. Also, font-family is already added to global CSS, so you can remove it here.

File: src/components/Success/Success.module.scss
Comment (ArnasDickus) [inline src/components/Success/Success.module.scss:103]: For performance, reason try to avoid nesting more when 3 levels.

File: src/components/Success/Success.module.scss
Comment (ArnasDickus) [inline src/components/Success/Success.module.scss]: The idea is this would be a popup that appears on top, so it doesn't need to have black background.

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for adding a PR.

1) There are some formatting issues. Try running `npm run format`
2) Also, try to use rem or em for accessibility instead of px. At least for sizes that are bigger than 10px.

File: src/components/Success/Success.module.scss
Comment (JimBratsos) [reply src/components/Success/Success.module.scss:103]: As I checked the nesting is exactly 3 levels if I am not wrong. If it does need fixing, please let me now

**Review by JimBratsos — COMMENTED** [review]
Fixed the mistakes, awaiting reply to the nesting comment. Waiting for approval

**Review by JimBratsos — COMMENTED** [review]
Fixed the mistakes, awaiting reply to the nesting comment. Waiting for approval

File: src/components/Success/Success.module.scss
Comment (ArnasDickus) [reply src/components/Success/Success.module.scss:103]: Yes, you right, on that one.

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, So you seem to have fixed all my issues. However, now there's a problem with the component. It seems background images repeat. Try adding `background-repeat: no-repeat` on background images.
background-repeat

**Review by JimBratsos — COMMENTED** [review]
All done, changes made

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello so your PR still have problems:
1) The component height is too big.
2) Success text is incorrectly placed.
text-wrong-too-big
[Success screen mockup](https://www.figma.com/file/KN4yf9KRGRLxPBlUy4VrixZS/Visual-Music?node-id=852%3A131)

**Review by JimBratsos — COMMENTED** [review]
Changes made as requested

File: src/components/Success/Success.module.scss
Comment (ArnasDickus) [inline src/components/Success/Success.module.scss]: align-items: center seems to do nothing

File: src/components/Success/Success.module.scss
Comment (ArnasDickus) [inline src/components/Success/Success.module.scss]: These lines seem to have no effect on code.
```
display: flex; 
justify-content: center; 
align-content: center;
```

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, so according to the [mockup](https://www.figma.com/file/KN4yf9KRGRLxPBlUy4VrixZS/Visual-Music?node-id=852%3A131). It's in the center. You seem to added on top instead of center. Without scrolling I can't see player bar. Also we don't actually know what avatar profile do, so you can remove it.

Looking at current situation, and considering that this is my 3rd review. I suggest to either start from scratch, try to avoid fixed width and height, or leave task to someone else.

sucess-screen

On mobile, you seem to created a bug of shorting playerbar. Bug appears only when success component is added.
mobile-success

**Comment by ArnasDickus** [comment]
Hello, @JimBratsos I just approved [PR #46 Error screen](https://github.com/zero-to-mastery/visual-music/pull/46). It might be useful to look into it.

**Review by JimBratsos — COMMENTED** [review]
Changes made, tested in most mobile phones and it's working smoothly. Thanks for sticking with me 
Desktop
Mobile View

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello [mockup](https://www.figma.com/file/KN4yf9KRGRLxPBlUy4VrixZS/Visual-Music?node-id=852%3A131)

The problem with this PR is:
Problem with mobile responsiveness. Also player bar bug.
sucess-not-responsive

On desktop not fully center:
on desktop not fully centered

My suggestion is just to copy [PR #46 css ](https://github.com/zero-to-mastery/visual-music/pull/46)

**Comment by JimBratsos** [comment]
The decision giving basically the same task to two different people created lots of confusion. Also, I do not know how it should be to be responsive in mobile. There is no template, or no information whether if it should cover the whole screen or not. It is supposed to be a popup otherwise it will be too small. Lastly, I cannot copy just the CSS, as my JS file and the error component is different. Again , assigning two people with the same thing, made lots of confusion. I will withdraw from this PR, as I cannot make it after all. Sorry

**Pull request closed without merging**
